docs/tutorials/tabular_prediction/tabular-deployment.md

```diff
@@ -134,6 +134,23 @@ path_clone_opt = predictor.clone_for_deployment(path=save_path_clone_opt)
 predictor_clone_opt = TabularPredictor.load(path=path_clone_opt)
 ```
 
+To avoid loading the model in every prediction call, we can persist the model in memory by:
+
+```{.python .input}
+predictor_clone_opt.persist_models()
+```
+
+In order to improve inference efficiency, we can call `.compile_models()` to automatically
+convert sklearn function calls into their onnx equivalence.
+Note that this is currently an experimental feature, which only improves RandomForest and TabularNeuralNetwork models.
+The compilation and inference speed acceleration require installation of `skl2onnx` and `onnxruntime` packages.
```

Comment:
Add:

To install supported verisons of these packages automatically, you can call  `pip install autogluon.tabular[skl2onnx]` on top of an existing AutoGluon installation, or `pip install autogluon.tabular[all,skl2onnx]` on a new AutoGluon installation.

Reply:
Added. Good call on adding installation guidance.